includes/class-wc-gateway-ppec-client.php

```diff
@@ -96,11 +96,11 @@ public function set_environment( $environment ) {
 	 * @return string
 	 */
 	public function get_endpoint() {
-		return sprintf(
+		return apply_filters( 'woocommerce_paypal_express_checkout_request_endpoint', sprintf(
```

Comment:
Just wondering if it'd be useful to pass some context to this filter, say `$this->_environment`

Reply:
👍 I had that originally and removed it without considering that it would actually be generally useful.